.github/workflows/generate.yml

```
@@ -0,0 +1,19 @@
+name: Generate CI
```

```suggestion
name: CI
```

"Generate CI" might not be the most appropriate name? Also maybe rename this file to `ci.yml`?

---

Done.